tests/sles4sap/hana_cluster.pm

```diff
@@ -78,6 +109,18 @@ sub run {
         sleep bmwqemu::scale_timeout(30);
         $self->do_hana_sr_register(node => $node1);
         sleep bmwqemu::scale_timeout(10);
+        if (check_var('ANGI', 'true')) {
+            # Setting up SAP HANA HA/DR providers
+            my $hadr_template = "angi_susHanaHADR_AIO.template";
+            assert_script_run "curl -f -v " . autoinst_url . "/data/sles4sap/$hadr_template -o /tmp/$hadr_template";
```

Comment:
No need for double quotes.
s/"curl -f -v "/'curl -f -v '/

Reply:
So join the doubles and singles in one assert_script_run?

Reply:
Sure, that is completely fine:
```
assert_script_run 'curl -f -v ' . autoinst_url . "/data/sles4sap/$hadr_template -o /tmp/$hadr_template";
```